site/content/docs/2.0/utilities/colors.md

```diff
@@ -85,7 +85,12 @@ Similar to the contextual text color classes, easily set the background of an el
 {{< /example >}}
 
 ## Background Gradient
+Use the `bg-gradient-*` class to apply a gradient background from a color on the bottom to transparent on the top.
+{{< example >}}
```

Comment:
The failing PR check is coming from linkinator:
```
> linkinator _site --recurse --skip "^(?!http://localhost)" --verbosity error

"file:/arizona-bootstrap-src/_site/docs/2.0/utilities/colors/index.html":521.438-521.441: error: No "p" element in scope but a "p" end tag seen.
```

This is what the rendered HTML looks on the review site (note the `<p></p>` getting wrapped around the example markup:
```


<h2 id="background-gradient">Background Gradient</h2>
<p>Use the <code>bg-gradient-*</code> class to apply a gradient background from a color on the bottom to transparent on the top.
  <div class="bd-example">
    <h3 class="bold p-card pt-8 bg-gradient-black text-white">.bg-gradient-black</h3>
  </div>
  <div class="highlight">
    <pre tabindex="0" class="chroma">
      <code class="language-html" data-lang="html">
        <span class="p">&lt;</span><span class="nt">h3</span> <span class="na">class</span><span class="o">=</span><span class="s">&#34;bold p-card pt-8 bg-gradient-black text-white&#34;</span><span class="p">&gt;</span>.bg-gradient-black<span class="p">&lt;/</span><span class="nt">h3</span><span class="p">&gt;</span>
      </code>
    </pre>
  </div>
</p>

```


```suggestion

{{< example >}}
```

Reply:
Oh dang, thanks @joeparsons!